aws-ts-netlify-cms-and-oauth/cms/infrastructure/index.ts

```diff
@@ -98,7 +98,7 @@ if (config.certificateArn === undefined) {
     }, { provider: eastRegion });
 
     const domainParts = getDomainAndSubdomain(config.targetDomain);
-    const hostedZoneId = aws.route53.getZone({ name: domainParts.parentDomain }, { async: true }).then(zone => zone.zoneId);
+    const hostedZoneId = aws.route53.getZoneOutput({ name: domainParts.parentDomain }, { async: true }).zoneId;
```

Comment:
These feel a bit funny, tbh, as they're trading off less idiomatic JS/TS in favor of the lifted property accessors on `Output`s

Reply:
Is there a problem with lifted property accessors, are they about to go away? What is idiomatic Pulumi? 

I forget to use them typically but I get review comments suggesting to use them:

https://github.com/pulumi/pulumi-hugo/pull/780#discussion_r735620716

Reply:
> What is idiomatic Pulumi?

This neatly summarizes my uneasiness with output functions in general. I don't see the need to use them unless I have an input that is itself an output--which is not the case in this example--as the "normal" versions are typically more idiomatic in the target language. In the past, we've tried to be as idiomatic as possible in the target language so that example code will look as familiar as possible to the end user.

As far as when to use lifted property accessors, I would say that any time the thing you're working with is already an output or is "naturally" an output (e.g. it is a resource output or the result of a call to an output function), then lifted property accessors are the way to go. If what you're working with is not naturally an output, it feels funny to force it to be an output just for the sake of lifted property accessors.

Reply:
Gotcha so a program that does not use outputs is preferable to one that does. I can revert.

Reply:
Reverted all similar changes.